src/portalbackend/PortalBackend.PortalEntities/PortalDbContext.cs

```diff
@@ -61,7 +61,7 @@ public PortalDbContext(DbContextOptions<PortalDbContext> options)
     public virtual DbSet<ApplicationChecklistEntryType> ApplicationChecklistTypes { get; set; } = default!;
     public virtual DbSet<AppSubscriptionDetail> AppSubscriptionDetails { get; set; } = default!;
     public virtual DbSet<AuditAppSubscriptionDetail20221118> AuditAppSubscriptionDetail20221118 { get; set; } = default!;
-    public virtual DbSet<AuditOffer20230119> AuditOffer20230119 { get; set; } = default!;
+    public virtual DbSet<AuditOffer20230406> AuditOffer20230406 { get; set; } = default!;
```

Comment:
The old db sets needs to be kept as well. Otherwise this will result in a drop table for the existing audit table which should not happen

```suggestion
    public virtual DbSet<AuditOffer20230119> AuditOffer20230119 { get; set; } = default!;
    public virtual DbSet<AuditOffer20230406> AuditOffer20230406 { get; set; } = default!;
```

Reply:
fixed